src/compensations/Compensations.tsx

```diff
@@ -34,6 +40,16 @@ const Compensations = ({ compensations, locations }: CompensationsProps) => {
     selectedDegree: "bachelor",
   });
 
+  const currentYearSalariesResult = useMemo(
+    () =>
+      salariesFromLocation(
+        currentYear,
+        selectedLocation,
+        compensations.salaries,
+      ),
+    [currentYear, selectedLocation, compensations.salaries],
+  );
+
   const updateSelectedDegree = (newDegree: Degree) => {
```

Comment:
Is it possible to update updateSelectedDegree() and updateExaminationYear() with one shared method for updating the formValues?

Reply:
`setFormState` is kind of the shared method, while `updateSelectedDegree` and `updateExaminationYear` are used to avoid inlining function bodies in the `<SalaryCalculator />` props